backend/src/main/java/com/google/sps/eventsender/EventSender.java

```diff
@@ -0,0 +1,206 @@
+
+  private static final String CREDENTIALS_FILE_PATH = "/credentials.json";
+
+  /**
+   * Creates an authorized Credential object.
+   *
+   * @param HTTP_TRANSPORT The network HTTP Transport.
+   * @return An authorized Credential object.
+   * @throws IOException If the credentials.json file cannot be found.
+   */
+  private static Credential getCredentials(final NetHttpTransport HTTP_TRANSPORT)
+      throws IOException {
+    // Load client secrets.
+    InputStream in = EventSender.class.getResourceAsStream(CREDENTIALS_FILE_PATH);
+    if (in == null) {
+      throw new FileNotFoundException("Resource not found: " + CREDENTIALS_FILE_PATH);
+    }
+    GoogleClientSecrets clientSecrets =
+        GoogleClientSecrets.load(JSON_FACTORY, new InputStreamReader(in));
+
+    // Build flow and trigger user authorization request.
+    GoogleAuthorizationCodeFlow flow =
+        new GoogleAuthorizationCodeFlow.Builder(HTTP_TRANSPORT, JSON_FACTORY, clientSecrets, SCOPES)
+            .setDataStoreFactory(new FileDataStoreFactory(new File(TOKENS_DIRECTORY_PATH)))
+            .setAccessType("offline")
+            .build();
+    LocalServerReceiver receiver = new LocalServerReceiver.Builder().setPort(8888).build();
+    return new AuthorizationCodeInstalledApp(flow, receiver).authorize("me");
+  }
+
+  /**
+   * TODO(): Convert from zonedTimeZone to DateTime
+   *
+   * @param meetingStartTime Time in which the meeting is designated to start at in the format
+   *     "yyyy-mm-dd"
+   * @param timezone String representing the Users Time Zone formatted as an IANA Time Zone Database
+   *     name, e.g. "Europe/Zurich".
+   * @return An EventDateTime with the input meeting time and input time zone.
+   */
+  public EventDateTime setMeetingStartTime(DateTime meetingStartTime, String timezone) {
+    return new EventDateTime().setDateTime(meetingStartTime).setTimeZone(timezone);
+  }
+
+  /**
+   * TODO(): Convert from zonedTimeZone to DateTime
+   *
+   * @param meetingEndTime Time in which the meeting is designated to end at in the format
+   *     "yyyy-mm-dd"
+   * @param timezone String representing the Users Time Zone formatted as an IANA Time Zone Database
+   *     name, e.g. "Europe/Zurich".
+   * @return An EventDateTime with the input meeting time and input time zone.
+   */
+  public EventDateTime setMeetingEndTime(DateTime meetingEndTime, String timezone) {
+    return new EventDateTime().setDateTime(meetingEndTime).setTimeZone(timezone);
+  }
+
+  /**
+   * @param participant1 Instance of the participant class representing an user in a successful
+   *     match
+   * @param participant2 Instance of a participant class representing a user in a successful match
+   * @return An array of EventAttendees with both of the participants emails pre set.
+   */
+  public EventAttendee[] createParticipantsArray(
+      com.google.sps.data.Participant participant1, com.google.sps.data.Participant participant2) {
```

Comment:
Would it be possible to add `com.google.sps.data.Participant` as an import so that this method header can be changed to Participant participant1 instead?